es-app/src/SystemScreenSaver.cpp

```diff
@@ -98,6 +98,7 @@ bool SystemScreenSaver::inputDuringScreensaver(InputConfig* config, Input input)
 				else if (is_start_input || is_select_game_input)
 				{
 					selectGame(is_start_input);
+					return input_consumed;
```

Comment:
I can't get it to crash on my end, but I believe you - it probably doesn't crash on my end out of luck. A question: what if we initialize input_consumed with (config->getMappedTo(input).size() == 0) and remove it from the condition below? Will it still crash, rather than returning here?

Reply:
Put it before, tested ok on my side.